drivers/SmartThings/zwave-switch/profiles/fibaro-double-switch.yml

```diff
@@ -14,52 +14,6 @@ components:
       version: 1
     categories:
     - name: Switch
-  - id: switch1
-    capabilities:
-    - id: switch
-      version: 1
-    - id: powerMeter
-      version: 1
-    - id: energyMeter
-      version: 1
-    categories:
-    - name: Switch
-  - id: button1
-    capabilities:
-    - id: button
-      version: 1
-    categories:
-    - name: RemoteController
-  - id: button2
-    capabilities:
-    - id: button
-      version: 1
-    categories:
-    - name: RemoteController
-  - id: button3
-    capabilities:
-    - id: button
-      version: 1
-    categories:
-    - name: RemoteController
-  - id: button4
-    capabilities:
-    - id: button
-      version: 1
-    categories:
-    - name: RemoteController
-  - id: button5
-    capabilities:
-    - id: button
-      version: 1
-    categories:
-    - name: RemoteController
-  - id: button6
-    capabilities:
-    - id: button
-      version: 1
-    categories:
-    - name: RemoteController
```

Comment:
I think we still want to use multiple button components, even though the DTH doesn't

Reply:
So, do you think the components should be in parent device profile or in child profile? Now events handled from parent device, so maybe it should stay her as it was?

Reply:
right but it uses the `buttonNumber` format which isn't supported anymore

Reply:
fixed